### PR TITLE
Document new model convenience functions

### PR DESCRIPTION
**Pull request by Mats-SX**

### CODE REVIEW

File: doc/modules/ROOT/pages/pipelines.adoc [hunk 1]
```diff
@@ -351,6 +356,9 @@ In addition to inheriting the methods common to all xref:model-object.adoc[model
 | predict_stream_estimate | G: Graph, +
                             config: **kwargs | Series      | https://neo4j.com/docs/graph-data-science/current/machine-learning/linkprediction-pipelines/predict/#algorithms-link-prediction-pipelines-predict-examples-estimate[Estimate predicting links between non-neighboring nodes of the input graph and streaming the results].
 | metrics                 | -                | Series      | Returns values for the metrics used when training, for this particular model.
+| best_parameters         | -                | Series      | Returns the training method parameters that performed best on the validation set.
+| node_property_steps     | -                | List[NodePropertyStep] | Returns a list of algorithms that the pipeline used for computing node properties before training.
```
Comment (FlorentinD) [inline doc/modules/ROOT/pages/pipelines.adoc:360]: NIT: you could extract these two items and share include them instead of copy-paste.

**Review by FlorentinD — APPROVED** [review]
Very convenient :)

File: doc/modules/ROOT/pages/pipelines.adoc (hunk 1 above)
Comment (Mats-SX) [reply doc/modules/ROOT/pages/pipelines.adoc:360]: yes that's a good idea! I will do that in next PR.

Comment (Mats-SX) [reply doc/modules/ROOT/pages/pipelines.adoc:360]: Done in #247

**Pull request merged**